streams/src/main/java/org/apache/kafka/streams/errors/LogAndContinueExceptionHandler.java

```diff
@@ -30,7 +29,7 @@
  * signals the processing pipeline to continue processing more records.
  */
 public class LogAndContinueExceptionHandler implements DeserializationExceptionHandler {
-    private static final Logger log = LoggerFactory.getLogger(StreamThread.class);
+    private static final Logger log = LoggerFactory.getLogger(LogAndContinueExceptionHandler.class);
```

Comment:
IIRC, using `StreamThread` is on purpose here (same below). @guozhangwang can you chime in?

If `StreamThread` is on purpose, we might want to add a comment.

Reply:
The original motivation is to maintain the information of which thread was hitting this error, but the current implementation does not maintain that any more.. I'm not sure if it was lost somewhere in previous commit or it is never the case. But this is what I've in mind originally:

```
        final String logPrefix = String.format("stream-thread [%s] ", threadClientId);
        final LogContext logContext = new LogContext(logPrefix);
        final Logger log = logContext.logger(LogAndContinueExceptionHandler.class);
```

Reply:
Are these classes public? Could we move them into `StreamThread`?

Reply:
Yes they are designed to be public.

Reply:
I see. There may not be a nice way to wire in the thread id from a static context. If we can't get the thread id, I'd suggest accepting the change here so that at least the message points to the right file.